content-repo/extra-docs/articles/SystemDiagnosticsHealthCheck.md

```diff
@@ -0,0 +1,135 @@
+---
+id: system-diagnostics-and-health-check
+title: Integrations and Incidents Health Check
```

Comment:
Should be change for the name of the use case System Diagnostics and Health Check.